crates/ark/src/interface.rs

```diff
@@ -774,6 +780,13 @@ impl RMain {
         if info.browser {
             match self.dap.stack_info() {
                 Ok(stack) => {
+                    if let Some(frame) = stack.first() {
+                        if let Some(ref env) = frame.environment {
```

Comment:
```suggestion
                    if let Some(ref env) = stack.first().map(|frame| frame.environment) {
```

Could probably drop a level of nesting with something like this? (Will need to be tweaked)

Reply:
I tried this before and tried again and could not get it to compile 😅 
Please feel free to tweak after I merge.